tests/all/js_globals/Number.rs

```diff
@@ -0,0 +1,127 @@
+#![allow(non_snake_case)]
+
+use super::project;
+
+
+#[test]
+fn to_locale_string() {
+    project()
+        .file("src/lib.rs", r#"
+            #![feature(proc_macro, wasm_custom_section)]
+
+            extern crate wasm_bindgen;
+            use wasm_bindgen::prelude::*;
+            use wasm_bindgen::js;
+
+            #[wasm_bindgen]
+            pub fn to_locale_string(this: &js::Number, locale: String) -> String {
+                this.to_locale_string(locale)
+            }
+        "#)
+        .file("test.ts", r#"
+            import * as assert from "assert";
+            import * as wasm from "./out";
+
+            export function test() {
+                let number = 1234.45;
+                assert.equal(wasm.to_locale_string(number, "de-DE"), "1,234.45");
+                assert.equal(wasm.to_locale_string(number, "en-US"), "1,234.45");
+                assert.equal(wasm.to_locale_string(number, "zh-Hans-CN-u-nu-hanidec"), "1,234.45");
```

Comment:
If you use this locale string for javascript in the browser you'll get chinese characters. Any idea why that doesn't work here?

Reply:
I assume it must be an issue with node.js then.

Reply:
This test fails for me, I get the chinese characters. Also the "de-DE" fails for me, I get "1.234,45".

Reply:
Hm, I wonder if it depends on some other system-specific things then. Perhaps we should just disable the test.